charts/rancher-monitoring/v0.0.7/charts/exporter-kube-state/templates/servicemonitor.yaml

```diff
@@ -48,3 +58,13 @@ spec:
       action: replace
       regex: (.+)
       replacement: $1
+    metricRelabelings:
```

Comment:
As we touched some metric relabelings rules, will they impact existing monitoring / alerting features? such as metrics of cluster monitoring in UI

Reply:
This change is to have a unique label so that we can selecting specific top10 metrics.
It adds additional labels to existing metrics and should not change current query result.
It also means that history metrics before upgrade might not be available in global monitoring dashboard due to missing these labels.